### ISSUE TITLE
SDA is recognized by Steam?

### ISSUE THREAD

**Issue by DerErzengel** [Jessecar96/SteamDesktopAuthenticator#1091]
I have heard rumors that steam can supposedly detect SDA and distribute community bans. Can this be true? 
Has there ever been such a case or something similar? 
I'm asking because I use SDA indirectly with an account that is quite valuable to me. (I used SDA Hash Value to transfer the time-based token to a secure Authenticator app).

**Comment by dyc3** [comment]
TL;DR: You don't have anything to worry about.

Technically, yes Valve could very easily detect usage of SDA for a number of reasons (for example, IIRC, SDA uses a hardcoded value for device id during setup, which they could use to correlate to usage of SDA). More importantly though, Valve doesn't have any motivation to ban people for using SDA or applications like it. I have never heard of someone getting banned for using SDA. IMHO, it's highly likely that anyone who claims that to be the case was doing something else that caused the ban, or had their account compromised and the malicious party got them banned.

But if it is something you are truly worried about, then you should start using the steam mobile app instead.
